locale/en_US/emails.po

```diff
@@ -25,7 +25,7 @@ msgstr ""
 "<br />\n"
 "Link: {$url}<br />\n"
 "<br />\n"
-"{$principalContactSignature}"
+"{$principalContactSignature} {$unsubscribeLink}"
```

Comment:
Should the new link be placed on a new line?

Reply:
I have put an `<hr />` in between.